simtools/util/names.py

```diff
@@ -748,3 +755,44 @@ def camera_efficiency_log_file_name(site, telescope_model_name, zenith_angle, la
     name += f"_{label}" if label is not None else ""
     name += ".log"
     return name
+
+
+def get_telescope_type(telescope_name):
+    """
+    Guess telescope type from name. Types are "LST", "MST", "SST", "SCT".
+    Parameters
+    ----------
+    telescope_name: str
+        Telescope name
+    Returns
+    -------
+    str
+        Telescope type.
+    """
+
+    _class, _ = split_telescope_model_name(telescope_name)
+    try:
+        if _class[0:3] in ("LST", "MST", "SST", "SCT"):
+            return _class[0:3]
+
+    except IndexError:
+        pass
+
+    return ""
+
+
+def get_corsika_telescope_data_dict():
+    io_handler = simtools.io_handler.IOHandler()
+    io_handler.set_paths(output_path="./simtools-output/", data_path="./data/")
+    file_data = collect_data_from_yaml_or_dict(
+        io_handler.get_input_data_file("parameters", "corsika_parameters.yml"), None
+    )
+    keys = ["corsika_sphere_center", "corsika_sphere_radius"]
```

Comment:
You should return the full dictionary, not just these keys.

Reply:
I disagree. I implemented this function to replace the previous manually defined dictionary and all info we need is passed here. in case at some point we need more info from this file we can implement it in this function or directly read the file again.

Reply:
My answer above should answer this as well I think